templates/openstack-userdata.sh.j2

```diff
@@ -13,3 +13,10 @@ su - ubuntu -c "cd ~/actions-runner && ./config.sh \
 
 # Run runner
 su - ubuntu -c "cd ~/actions-runner && /home/ubuntu/actions-runner/run.sh"
+<<<<<<< HEAD
+
+# cleanup
+sleep 2
+ps -aux
+=======
+>>>>>>> c57beb0daae5a7c242a7eb89409db8b6d815029b
```

Comment:
Should this be here?

Reply:
Fixed, seems like VS-Code wasn't quite able to check this conflict.